go.mod

```diff
@@ -28,6 +28,9 @@ require (
 
 replace (
 	k8s.io/api => k8s.io/api v0.0.0-20190704095032-f4ca3d3bdf1d
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190704094733-8f6ac2502e51
 	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v0.2.0
+
+	// can be a tag when https://github.com/kubernetes-sigs/controller-runtime/commit/524b614e5a34c80432744688e1a90a0f3b155780 is in a release
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.2.0-beta.1.0.20190911194436-524b614e5a34
+	k8s.io/client-go => k8s.io/client-go 78d2af792babf2dd937ba2e2a8d99c753a5eda89 // v12.0.0
```

Comment:
Can we use the k8s version tag rather than the client-go version tag here?
I would also expect the reference in the `require` block to match this as well.